docs/guides/kubernetes/deploy-and-manage-a-cluster-with-linode-kubernetes-engine-a-tutorial/index.md

```diff
@@ -177,6 +177,41 @@ You can use the Linode Cloud Manager to modify a cluster's existing node pools b
 
 {{< content "kubernetes-edit-remove-node-pools-shortguide" >}}
 
+### Configure Cluster Autoscaling
+
+In Kubernetes, Cluster Auto-Scaling refers to a method by which Kubernetes users can configure their cluster to automatically scale the amount of physical nodes available in a node pool up and down as hardware needs of the the pool increase or decrease. While this feature can be applied manually using resources like the [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) provided by Kubernetes, LKE can manage this potential automatically through the Cloud Manager and the [Linode API](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/).
+
+The LKE autoscaler will only apply changes when the following conditions are met:
+
+- If Pods are unschedulable due to an insufficient number of Nodes in the Node Pool, the auto-scaler will increase the number of physical nodes to the amount  required.
+
+- If Pods are able to be scheduled on less Nodes than are currently available in the Node Pool,  Nodes will be drained and removed automatically. Pods on drained nodes will be immediately rescheduled on pre-existing nodes. The Node Pool will be decreased to match only the needs of the current workload.
+
+LKE Autoscaling is configured for individual Node Pools directly through the Linode Cloud Manager.
+
+1. To Enable cluster autoscaling, access the [cluster's details page](/docs/kubernetes/deploy-and-manage-a-cluster-with-linode-kubernetes-engine-a-tutorial/#access-your-cluster-s-details-page).
+
+1. Click the **Autoscale Pool** option at the top-left of each entry in the **Node Pools** section. The Autoscaling menu will appear.
+
+    ![Autoscale Pool Button](autoscale-pool-button.png "Kubernetes cluster pool autoscaler button.")
+
+1. If the Autoscaler is currently disabled, select the autoscaler switch toggle to turn the feature on.
+
+    ![Autoscaler Switch Toggle](autoscaler-switch-toggle.png "LKE autoscaler toggle switch.")
+
+1. Once the Autoscaler is enabled, the **Minimum** `Min` and **Maximum** `Max` fields can be set. Both the Minimum and Maximum field can be any number between 1 and 99. Each number represents a single Node in the node pool.
```

Comment:
Do we need to be more specific about minimum and maximum being a count of nodes? Also, `Each number represents a single Node in the node pool`: this is awkward, how could "3 min / 5 max" represent a single node? Shouldn't it be a target number of nodes in the node pool?